### PR TITLE
 [MIG] procurement_purchase_no_grouping: Migration to 11.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
* Squash together adjacent translation commits
* Fix tests
* Generate new README style

**Review by pedrobaeza — APPROVED** [review]
Points fixed

@carlosdauden please review my additions

**Comment by andreampiovesana** [comment]
commit?

**Comment by pedrobaeza** [comment]
See the last one

File: procurement_purchase_no_grouping/views/product_category_view.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (anandkansagra) [inline procurement_purchase_no_grouping/views/product_category_view.xml:1]: Remove `encoding="utf-8"`

File: procurement_purchase_no_grouping/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (anandkansagra) [inline procurement_purchase_no_grouping/models/__init__.py]: Remove `# -*- coding: utf-8 -*-`

**Review by anandkansagra — CHANGES_REQUESTED** [review]

File: procurement_purchase_no_grouping/models/__init__.py (hunk 2 above)
Comment (carlosdauden) [reply procurement_purchase_no_grouping/models/__init__.py]: Ok

File: procurement_purchase_no_grouping/views/product_category_view.xml (hunk 1 above)
Comment (carlosdauden) [reply procurement_purchase_no_grouping/views/product_category_view.xml:1]: Are you sure that this isn't needed?

Comment (pedrobaeza) [reply procurement_purchase_no_grouping/views/product_category_view.xml:1]: This one is totally needed in XML files. It's on Python files where you don't need encoding from Python 3

Comment (anandkansagra) [reply procurement_purchase_no_grouping/views/product_category_view.xml:1]: Thanks @pedrobaeza ,
LGTM

File: procurement_purchase_no_grouping/models/__init__.py (hunk 2 above)
Comment (pedrobaeza) [reply procurement_purchase_no_grouping/models/__init__.py]: Done

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**